samples/mount_sample/mount.go

```diff
@@ -111,7 +111,7 @@ func main() {
 	runtime.GOMAXPROCS(2)
 
 	// Grab the file to signal when ready.
-	readyFile, err := getReadyFile()
+	readyFile, err := os.Create("/tmp/ready_file")
```

Comment:
@dotslash Can you explain a bit on the reasoning behind this change please?

Reply:
Undo the changes in this file. I was debugging and did not (care to) understand how getReadyFile works